tests/cli/models/test_tags.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from cg_hermes.models.tags import TagMap
+
+
+def test_instantiate_tag_map(tag_map_raw: dict):
+    """
+    Tests instantiating model."""
+    # GIVEN a dictionary with a tag map
+
+    # WHEN instantiating an object
+    tag_map = TagMap.model_validate(tag_map_raw)
+
+    # THEN assert that it was successfully created
+    assert isinstance(tag_map, TagMap)
+
+
+def test_tag_map_tags_validator(tag_map_raw: dict):
+    """
+    Tests instantiating model."""
```

Comment:
any reason for this two-line docstring? resolve if intended

Reply:
Fixed